Add a section on creating a run config. (#463)
OpenLiberty/liberty-tools-intellij: docs/user-guide.md

   - [Gradle-built applications](#gradle-built-applications)
- [Stop your application](#stop-your-application)
- [Debug your application](#debug-your-application)

> - [Create a new Liberty Run/Debug Configuration](#create-a-new-liberty-rundebug-configuration)

Reviewer: ```suggestion
- [Create a Liberty Run/Debug Configuration](#create-a-liberty-rundebug-configuration)
```

Author: Yes.

![Liberty debugger attached](images/liberty-debugger-attached.png)

> ## Create a new Liberty Run/Debug Configuration

Author: Ok

> The Liberty Tools plugin contributes a new type of Run/Debug configuration. Select the **Run -> Edit Configurations** menu to open the **Edit Run/Debug configurations** dialog. Click **+** and select Liberty from the menu. 

Reviewer: ```suggestion
The Liberty Tools plugin contributes a its own Run/Debug configuration. Select the **Run -> Edit Configurations** menu to open the **Edit Run/Debug configurations** dialog. Click **+** and select Liberty from the menu. 
```

![Liberty new configuration](images/liberty-new-config.png)

> For more information see the [IDE documentation](https://www.jetbrains.com/help/idea/run-debug-configuration.html).

Reviewer: ```suggestion
For more information, see [Run/debug configurations](https://www.jetbrains.com/help/idea/run-debug-configuration.html) in the Intellij IDEA documentation.
```

Author: Also ok.
